EstateReporting/Controllers/ReportingController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EstateReporting.Controllers
+{
+    using System.Diagnostics.CodeAnalysis;
+    using System.Threading;
+    using BusinessLogic;
+    using DataTransferObjects;
+    using Factories;
+    using Microsoft.AspNetCore.Mvc;
+    using Models;
+
+    [Route(ReportingController.ControllerRoute)]
+    [ApiController]
+    [ExcludeFromCodeCoverage]
+    public class ReportingController : ControllerBase
+    {
+        private readonly IReportingManager ReportingManager;
+
+        private readonly IModelFactory ModelFactory;
+
+        public ReportingController(IReportingManager reportingManager, IModelFactory modelFactory)
+        {
+            this.ReportingManager = reportingManager;
+            this.ModelFactory = modelFactory;
+        }
+
+        #region Others
+
+        /// <summary>
+        /// The controller name
+        /// </summary>
+        public const String ControllerName = "reporting";
```

Comment:
Codacy found an issue: [Change this constant to a 'static' read-only property.](https://app.codacy.com/manual/stuart_ferguson1/EstateReporting/pullRequest?prid=6325301)